.mergify.yml

```diff
@@ -1,5 +1,14 @@
 pull_request_rules:
-  - name: merge on CI success and 2 approvals
+  - name: instruct bors to merge dependabot PRs with passing tests
+    conditions:
+      - "status-success=ci/circleci: check-format"
+      - "status-success=ci/circleci: stable-build"
```

Comment:
This should be beta (as aligned with comit-rs and the branch rules of this repo).